tests/testCoaddApCorrMap.py

```diff
@@ -20,26 +20,19 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-
-"""
-Tests for CoaddApCorr code
-
-Run with:
-   python CoaddApCorr.py
-"""
-
+from __future__ import absolute_import, division, print_function
 import os
-import numpy
+import numpy as np
 import unittest
 
-import lsst.utils.tests as utilsTests
 import lsst.afw.geom as afwGeom
 import lsst.afw.math as afwMath
 import lsst.afw.table as afwTable
 import lsst.afw.image as afwImage
 import lsst.afw.coord as afwCoord
-import lsst.meas.algorithms as measAlg
 from lsst.afw.geom.polygon import Polygon
+import lsst.meas.algorithms as measAlg
+import lsst.utils
 
```

Comment:
I think this was because of the lsst.utils.getPackageDir... call later on, but if we use __file__ instead then this could be lsst.utils.tests